### ISSUE TITLE
[BUG] H026 false positive if id/class appear within template syntax

### ISSUE THREAD

**Issue by jayvdb** [Riverside-Healthcare/djLint#89]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: Windows
 - Python Version: 3.7
 - djLint Version: 0.5.6

## Issue
H026 is occurring if `id` occurs anywhere in the attributes, including in the value of another attribute.
```html
<a href="{% url 'foo:bar' id %}">..</a>
```

## How To Reproduce
See above
